scripts/format.sh

```diff
@@ -16,13 +16,14 @@ source scripts/bash_utils.sh
 # Python setup
 py=$(setup_python)
 install_py_packages $py
-python_sources=$(LANG=C find scripts src/bindings/python -type f -exec file {} \; | grep -F "Python script" | sed 's/:.*//')
+python_sources=$(LANG=C find scripts src/bindings/python -type f ! -name "*.md" -exec file {} \; | grep -F "Python script" | sed 's/:.*//')
```

Comment:
`file` interprets a markdown file as `Python script` if it contains explicitly called out python code blocks :roll_eyes: which then `black` chokes on.